fimsauthz-api/swagger.yaml

```diff
@@ -16,20 +16,32 @@ securityDefinitions:
   application:
     type: oauth2
     # Should be updated to the Token URL of your OAuth provider.
-    tokenUrl: https://thisserver.com/oauth/token
+    tokenUrl: https://utm.defined.host/oauth/token
     flow: application
     scopes:
-      'utm.nasa.gov_read.constraint': Subject can read UTM constraint data
-      'utm.nasa.gov_write.constraint': Subject can read, create, and update UTM constraint data. This means that the Subject can define areas that restrict other operations.
-      'utm.nasa.gov_read.min_data': Subject can read data which has been minimized for example, by redacting some properties
-      'utm.nasa.gov_write.operation': Subject can read, create, and update operational data such as USS Instance data
+      'utm.nasa.gov_read.fimsadmin': Subject can read data internal to FIMS.
+      'utm.nasa.gov_write.fimsadmin': Subject can read, create, and update data internal to FIMS.
       'utm.nasa.gov_read.operation': Subject can read operational data such as USS Instance data
-      'utm.nasa.gov_write.message': Subject can read, create, and update message data such as UTM Message and Negotiations
+      'utm.nasa.gov_write.operation': Subject can read, create, and update operational data such as USS Instance data
       'utm.nasa.gov_read.message': Subject can read message data such as UTM Message and Negotiations
+      'utm.nasa.gov_write.message': Subject can read, create, and update message data such as UTM Message and Negotiations
+      'utm.nasa.gov_read.publicsafety': Subject can read operations that are designated public safety operations.
       'utm.nasa.gov_write.publicsafety': Subject can read, create, and update operations that are designated public safety operations.
+      'utm.nasa.gov_read.constraint': Subject can read UTM constraint data
+      'utm.nasa.gov_write.constraint': Subject can read, create, and update UTM constraint data. This means that the Subject can define areas that restrict other operations.
+      'utm.nasa.gov_read_min.uvin': Subject can read redacted vehicle registration data.
```

Comment:
do we need something similar for operations and messages?  Are there use cases wherein the requestor should only get redacted/minimal data response for operations/messages?

Reply:
I am in favor of NOT having min-data.   I left read_min.uvin in because vehicle API wrote it in, but I would lobby to have vehicle take it out. 

I could be wrong but my quick look shows that  popular APIs  only have oauth2 read and write. 

I think redaction should not be implemented in the security layer, rather it is  business decision about how to react for an Anonymous principal, or for a principle that doesn't have sufficient oauth2 scopes.

The internet searches i did were something like "best practices redaction organizational policy vs. security roles"